intelmq-manager/php/config.php

```diff
@@ -29,7 +29,7 @@
         'positions' => '/opt/intelmq/etc/manager/positions.conf',
     );
     # get paths from intelmqctl directly if it works
-    $proc = proc_open($c . "--type json debug --get-paths", [
+    $proc = proc_open($c . " --type json debug --get-paths", [
```

Comment:
Thanks for reporting that bug!
However, I propose to instead use
```python
sprintf($CONTROLLER_JSON, "debug --get-paths")
```
as command to be in line with the other command-compositions in the controller file.